Utilities/Hooks/prepare-commit-msg

```diff
@@ -39,8 +39,9 @@ instructions='#\
 #   STYLE:  - no logic impact (indentation, comments)\
 #   WIP:    - Work In Progress not ready for merge\
 # To have an automatic link created between this patch set\
-# and the issue referenced at issues.itk.org, simply\
-# add ITK-#### in somewhere in the commit message.
+# and the issue referenced at\
+# https://github.com/InsightSoftwareConsortium/ITK/issues,\
```

Comment:
We should continue to use `issues.itk.org`, since it is easier to remember and type and in case the issue tracker ever changes again. `issues.itk.org` redirects to `https://github.com/InsightSoftwareConsortium/ITK/issues`.